end-to-end-test/local/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2016 The Hyve B.V.
```

Comment:
let's use the now official cbioportal/cbioportal image here

Reply:
Docker solution was set to the new docker solution.